adapters/eth_tx.go

```diff
@@ -26,7 +25,7 @@ type EthTx struct {
 	FunctionSelector models.FunctionSelector `json:"functionSelector"`
 	DataPrefix       hexutil.Bytes           `json:"dataPrefix"`
 	DataFormat       string                  `json:"format"`
-	GasPrice         *big.Int                `json:"gasPrice" gorm:"type:varchar(255)"`
+	GasPrice         *models.Big             `json:"gasPrice" gorm:"type:varchar(255)"`
```

Comment:
Might be candidate for a more appropriate type? `NUMERIC` ?

Reply:
Assuming you mean sql type, numeric isn't supported by sqlite 😞  : https://www.tutorialspoint.com/sqlite/sqlite_data_types.htm

Reply:
Ok, I'll try